girder_annotation/docs/annotations.rst

```diff
@@ -278,21 +280,22 @@ a ``2x2`` affine matrix.
 
 ::
 
-    {
-        "type": "imageoverlay",           # Exact string. Required
-        "girderId": <girder image id>,    # 24-character girder id pointing
-                                          # to a large image object. Required
-        "opacity": 1,                     # Default opacity for the overlay. Defaults to 1. Optional
-        "transform": {                    # Object specifying additional overlay information. Optional
-            "xoffset": 0,                 # How much to shift the overlaid image right.
-            "yoffset": 0,                 # How much to shift the overlaid image down.
-            "matrix": [                   # Affine matrix to specify tranformations like scaling,
-                                          # rotation, or shearing.
-                [1, 0],
-                [0, 1]
-            ]
-        }
+  {
+    "type": "imageoverlay",            # Exact string. Required
+    <id, label>                        # Optional general shape properties
```

Comment:
Does it make sense to keep consistent with other element types here?

```suggestion
    <id, label, group, user>           # Optional general shape properties
```

Reply:
Yes.  In my mind I had updated that line, but not in reality.